ntbs-service/Pages/ContactDetails/Edit.cshtml

```diff
@@ -112,6 +112,30 @@
                 </nhs-form-group>
             </validate-input>
             
+            @if (Model.ContactDetails.CaseManagerTbServices?.Any() ?? false)
+            {
+                <nhs-form-group nhs-form-group-type="@jobTitleState"
+                                id="@($"{nameof(Model.ContactDetails)}-{nameof(Model.ContactDetails.IsCaseManager)}")">
+                    <label nhs-label-type="Standard" asp-for="ContactDetails.JobTitle">
+                        Is a case manager?
+                    </label>
+                    <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
+                        <div class="nhsuk-radios__item">
+                            <input asp-for="ContactDetails.IsCaseManager" id="iscasemanager-yes" class="nhsuk-radios__input" type="radio" value="true" disabled="@Model.UserHasNotification">
```

Comment:
I'm slightly wary of disabling these options whenever the user has a notification - if a user somehow ended up in an invalid state where they had a notification but weren't a case manager then there wouldn't be a way to correct this.

Reply:
We could only disable the options when their choice is "Yes" and they have a notification?

Reply:
Yeah, I think that would be safer